templates/api.yaml

```diff
@@ -1098,7 +1103,6 @@ Resources:
     Properties:
       Handler: handlers.process_handler
       CodeUri: ../backend/lambdas/queue/
-      MemorySize: 512
```

Comment:
This is an anomaly for the API handler. We used to read the whole queue during this phase and when we switched to asynchronous query planning (https://github.com/awslabs/amazon-s3-find-and-forget/pull/239) we forgot to update this value back to 128.